src/main/java/school/lemon/changerequest/java/introduction/hw2/Task2.java

```diff
@@ -3,6 +3,19 @@
 
 public class Task2 {
     public static String firstTwo(String s) {
-        return "";
+
+        if (s == null)
+            return null;
+        StringBuilder shortstr = new StringBuilder(s);
+        if (s.length() < 2)
+            return s;
+
+
+        else {
+            shortstr.setLength(2);
+            String substring = shortstr.substring(0);
+            return substring;
+        }
+
```

Comment:
It's quite uncommon way to solve this task.
Actually, there is no need for StringBuilder here - you can just use `substring` method of `String`.

Also, you can move length check to null-check, so that you'll check string for null and for `length<2` and just return `s`.